packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/views/FormViewControllerIntegrationTests.java

```diff
@@ -0,0 +1,229 @@
+import java.util.function.Consumer;
+
+import org.eclipse.sirius.components.collaborative.dto.CreateChildInput;
+import org.eclipse.sirius.components.collaborative.dto.CreateChildSuccessPayload;
+import org.eclipse.sirius.components.collaborative.forms.dto.EditTextfieldInput;
+import org.eclipse.sirius.components.collaborative.forms.dto.FormRefreshedEventPayload;
+import org.eclipse.sirius.components.core.api.SuccessPayload;
+import org.eclipse.sirius.components.forms.AbstractWidget;
+import org.eclipse.sirius.components.forms.Textarea;
+import org.eclipse.sirius.components.forms.tests.assertions.FormAssertions;
+import org.eclipse.sirius.components.forms.tests.graphql.EditTextfieldMutationRunner;
+import org.eclipse.sirius.components.forms.tests.navigation.FormNavigator;
+import org.eclipse.sirius.web.AbstractIntegrationTests;
+import org.eclipse.sirius.web.application.views.details.dto.DetailsEventInput;
+import org.eclipse.sirius.web.data.StudioIdentifiers;
+import org.eclipse.sirius.web.tests.graphql.CreateChildMutationRunner;
+import org.eclipse.sirius.web.tests.graphql.DetailsEventSubscriptionRunner;
+import org.eclipse.sirius.web.tests.services.api.IGivenCommittedTransaction;
+import org.eclipse.sirius.web.tests.services.api.IGivenInitialServerState;
+import org.eclipse.sirius.web.tests.services.representation.RepresentationIdBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlConfig;
+import org.springframework.transaction.annotation.Transactional;
+
+import graphql.execution.DataFetcherResult;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+/**
+ * Integration tests of From View controllers.
+ *
+ * @author frouene
+ */
+@Transactional
+@SuppressWarnings("checkstyle:MultipleStringLiterals")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class FormViewControllerIntegrationTests extends AbstractIntegrationTests {
+
+    @Autowired
+    private IGivenInitialServerState givenInitialServerState;
+
+    @Autowired
+    private IGivenCommittedTransaction givenCommittedTransaction;
+
+    @Autowired
+    private CreateChildMutationRunner createChildMutationRunner;
+
+    @Autowired
+    private DetailsEventSubscriptionRunner detailsEventSubscriptionRunner;
+
+    @Autowired
+    private RepresentationIdBuilder representationIdBuilder;
+
+    @Autowired
+    private EditTextfieldMutationRunner editTextfieldMutationRunner;
+
+    @BeforeEach
+    public void beforeEach() {
+        this.givenInitialServerState.initialize();
+    }
+
+    @Test
+    @DisplayName("Given a view, when a from description is created, then it is created properly")
```

Comment:
Honestly, the whole file should probably be deleted, this test does not match what is expected in our tests:

- It uses tons of changes during the test to setup the real initial state which should be done in the SQL file
- The core part of the test is testing the edition and subsequent refresh of a form description (not its creation) thanks to the details view
- I don't see the link with diagrams


Tests should be organized in three parts:

- GIVEN: the initial state configured in the sql file
- WHEN: ONE change to execute the behavior to be tested
- THEN: assertions on the result


The simple fact that a second method is needed to initialize programmatically the state of the test thanks to multiple inputs is a big red flag.